Jenkinsfile

```diff
@@ -0,0 +1,144 @@
+@Library(value='pipeline-library@master', changelog=false)
+import com.genesys.jenkins.Service
+ 
+final serviceBuild = new Service()
+
+pipeline {
+
+    agent any
+
+    tools {
+        maven 'Maven 3.5.3'
+        jdk 'OpenJDK 1.8.0 (latest)'
+    }
+
+    environment {
+        NODE_VERSION = '11.15.0'
+        SNYK_TOKEN = credentials("PureCloudSnykToken")
+    }
+
+    stages {
+        stage("pre-build") {
+            steps {
+                copyArtifacts(projectName: 'build-github-swagger-codegen-sdkv1', filter: 'modules/swagger-codegen-cli/target/swagger-codegen-cli.jar', target: 'swagger-codegen', fingerprintArtifacts: true);
+                dir('platform-client-sdk-internal-config') {
+                    git url: 'git@bitbucket.org:inindca/platform-client-sdk-internal-config.git'
```

Comment:
Should use a variable for this repo too. Generally keep these things private

Reply:
Fixed. Changed to use SCM variable